rules/linux/execution_remote_code_execution_via_postgresql.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2022/06/20"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/06/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for suspicious activities that may indicate an attacker attempting to execute arbitrary code within 
+a PostgreSQL environment. Attackers execute code via PostgreSQL to exploit vulnerabilities, achieve remote command 
+execution, perform SQL injection attacks, and facilitate post-exploitation activities for unauthorized access and 
+malicious actions.
```

Comment:
> Attackers execute code via PostgreSQL to exploit vulnerabilities, achieve remote command 
execution, perform SQL injection attacks, and facilitate post-exploitation activities for unauthorized access and 
malicious actions.

Maybe we could change this to something like this:

> Attackers can execute code via PostgreSQL as a result of exploiting vulnerabilities, such as remote command execution and SQL injection attacks, which can result in unauthorized access and malicious actions.

Reply:
Ohhh, this is local? Would make sense to change this one to cover suspicious childs?